Farmland.py

```diff
@@ -1,205 +0,0 @@
-#This contains the two classes of BarrenLand and FertileLand
-
-import numpy.matlib
-import numpy as np
-
-#class BarrenLand holds information about the barren land,
-class BarrenLand:
-
-    #input is the string with four integers seperated by a space
-    def __init__(self, LowerLeftVertex=-1, UpperRightVertex=-1, stringVertex= '',fieldwidth = 400,fieldheight = 600):
-        if stringVertex != '':
-            # vertex list seperates the string into a list of 4 integers
-            self.vertexList = list(map(int, stringVertex.split()))
-        else :
-            self.vertexList = [LowerLeftVertex[0],LowerLeftVertex[1], UpperRightVertex[0], UpperRightVertex[1]]
-
-        if self.vertexList[0] < 0 or self.vertexList[1] < 0 or self.vertexList[2] >= fieldwidth or self.vertexList[3] >=fieldheight:
-            raise ValueError("The input of the vertex is out of bounds!")
-
-        #saves the vertex locations as attributes

```

Comment:
Are you able to create a loop to assign these values to `self.vertexList[n]`? There is a coding principle called 'DRY' - meaning, "Don't Repeat Yourself"